core-web/libs/dotcms-scss/angular/dotcms-theme/components/_button.scss

```diff
@@ -415,30 +414,12 @@
     background-color: rgba(255, 64, 129, 0.16);
 }
 
-.p-button.p-button-danger:enabled:focus,
-.p-buttonset.p-button-danger > .p-button:enabled:focus,
-.p-splitbutton.p-button-danger > .p-button:enabled:focus {
-    background: rgba(211, 47, 47, 0.76);
-}
-
-.p-button.p-button-danger:enabled:active,
-.p-buttonset.p-button-danger > .p-button:enabled:active,
-.p-splitbutton.p-button-danger > .p-button:enabled:active {
-    background: rgba(211, 47, 47, 0.68);
-}
-
 .p-button.p-button-danger.p-button-text:enabled:focus,
 .p-buttonset.p-button-danger > .p-button.p-button-text:enabled:focus,
 .p-splitbutton.p-button-danger > .p-button.p-button-text:enabled:focus {
     background: rgba(211, 47, 47, 0.12);
 }
 
-.p-button.p-button-danger.p-button-text:enabled:active,
-.p-buttonset.p-button-danger > .p-button.p-button-text:enabled:active,
-.p-splitbutton.p-button-danger > .p-button.p-button-text:enabled:active {
-    background: rgba(211, 47, 47, 0.16);
-}
-
```

Comment:
Can't be removed... we need to unity because the other selector have another props.